examples/with-universal-configuration/env-config.js

```diff
@@ -0,0 +1,5 @@
+const prod = process.env.NODE_ENV === 'production'
+
+module.exports = {
+  'MY_ENV_VARIABLE': prod ? 'production' : 'development'
```

Comment:
Shall we use a bit more realistic example. For an example, let's create a variable called `BACKEND_URL`.

Use different URLs based other env.

In the component, just say loading data from: BACKEND_URL